### PR TITLE
Tell the world that arXiv labs is open for business again

### PR DESCRIPTION
**Pull request by cbf66**
Change links to Jira Service Desk Labs portal for new proposals, support, and questions.

### CODE REVIEW

**Comment by bdc34** [comment]
The preview is here: https://storage.googleapis.com/arxiv-docs-prs/cbf/restart-labs/labs/index.html

**Comment by SBBCornell** [comment]
I just put through a proposal, and moderated it in Jira. It looks good (I added a "rejected" status to the workflow, Charles. Inspired by my amazing Labs proposal). In addition to internal comments, I see there is also an "approve" feature in Jira which might be useful. By default it just switches it to 'in progress' status, but we could use it for some other status or to notify the user, presumably.

File: labs/criteria.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # arXivLabs Criteria
 
-_*arXiv Labs is temporarily on pause while arXiv is focused on stability and moving to the cloud. Please check back in six months.*_
+_arXiv Labs is back from vacation._ As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```
Comment (SBBCornell) [inline labs/criteria.md:3]:
```suggestion
As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```

Comment (SBBCornell) [reply labs/criteria.md:3]: The NSF grand folks will likely not know it was offline for a bit. Why draw attention to it?

File: labs/index.md [hunk 2]
```diff
@@ -130,8 +130,10 @@ arXiv welcomes anyone, from single individuals to large companies, to contribute
 
 <!-- <a href="project-proposal" class="button-fancy">Click to submit your idea <span> </span></a> -->
 
-_*arXiv Labs is temporarily on pause while arXiv is focused on stability and moving to the cloud. Please check back in six months.*_ In the meantime, you can learn more about the [criteria](criteria) for arXiv Labs projects.
+_arXiv Labs is back from vacation._ As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```
Comment (SBBCornell) [inline labs/index.md:133]:
```suggestion
As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```

File: labs/project-proposal.md [hunk 3]
```diff
@@ -30,13 +30,15 @@ blockquote {
 }
 </style>
 
-_*arXiv Labs is temporarily on pause while arXiv is focused on stability and moving to the cloud. Please check back in six months.*_
+# Submit your project to arXivLabs
+
+_arXiv Labs is back from vacation._ As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```
Comment (SBBCornell) [inline labs/project-proposal.md:35]:
```suggestion
As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```

File: source/labs/criteria.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # arXivLabs Criteria
 
-_*arXiv Labs is temporarily on pause while arXiv is focused on stability and moving to the cloud. Please check back in six months.*_
+_arXiv Labs is back from vacation._ As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```
Comment (SBBCornell) [inline source/labs/criteria.md:3]:
```suggestion
As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```

File: source/labs/project-proposal.md [hunk 5]
```diff
@@ -1,15 +1,11 @@
 # Submit your project to arXivLabs
 
-_*arXiv Labs is temporarily on pause while arXiv is focused on stability and moving to the cloud. Please check back in six months.*_
+_arXiv Labs is back from vacation._ As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```
Comment (SBBCornell) [inline source/labs/project-proposal.md:3]:
```suggestion
As of December 2022 we are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.
```

**Review by SBBCornell — APPROVED** [review]
Looks good. I made the same wording suggestion across a few pages.

**Comment by jimentwood** [comment]
I wonder if the status in that case should be 'declined'. I think _customers_ can see the status of their tickets.
________________________________

File: labs/criteria.md (hunk 1 above)
Comment (cbf66) [reply labs/criteria.md:3]: Well, this isn't only for NSF. What if it just said "We are, once again, open for ..."? And take out "As of December 2022 ..."

**Comment by cbf66** [comment]
> I wonder if the status in that case should be 'declined'. I think _customers_ can see the status of their tickets.
 
@jimentwood -- yeah, I think the current wording is "rejected". I'm pretty sure this can be customized. Feel free to change it to something more polite. Also, consider if we need more states -- maybe states that would show the customer that the ticket is passing through various levels of review?

(Oh my, we're now making this a mini-submission system ...)

**Comment by jimentwood** [comment]
Maybe just delete that section?
I think most people coming to that page won't care if there was a period where we weren't accepting proposals.
________________________________

File: source/labs/project-proposal.md (hunk 5 above)
Comment (afromme) [reply source/labs/project-proposal.md:3]: I would take it further, and just say:
We are open for reviewing new proposals. Please <a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/6">use our support portal</a> to fill out a proposal, or ask questions.

Comment (afromme) [reply source/labs/project-proposal.md:3]: I might also suggest adding a time frame, as in:
Evaluation of labs proposals can take up to 6 months.

**Review by jimentwood — APPROVED** [review]

**Review by SBBCornell — COMMENTED** [review]
The wording looks good to me, @afromme!

**Review by afromme — APPROVED** [review]

**Pull request merged**